src/nft/components/profile/view/ViewMyNftsAsset.tsx

```diff
@@ -126,7 +126,8 @@ export const ViewMyNftsAsset = ({
             text={getUnsupportedNftTextComponent(asset)}
             placement="bottom"
             offsetX={0}
-            offsetY={-100}
+            offsetY={-60}
```

Comment:
why is this -60 when the other one is -50?

Reply:
cuz it looks weird when its 50 😅 too close to the bottom